Dockerfile

```diff
@@ -29,5 +29,5 @@ COPY --from=builder /go/src/github.com/oauth2-proxy/oauth2-proxy/oauth2-proxy /b
 COPY --from=builder /go/src/github.com/oauth2-proxy/oauth2-proxy/jwt_signing_key.pem /etc/ssl/private/jwt_signing_key.pem
 
 USER 2000:2000
-
+EXPOSE 4180
```

Comment:
- please have one empty line before and after this command. that is how the rest of the dockerfile is written

Reply:
handled